.travis.yml

```diff
@@ -8,17 +8,10 @@ cache:
   yarn: true
   directories:
   - node_modules
-  - /home/travis/.pyenv_cache
+notifications:
+  email: false
 install:
   - yarn install
-before_install:
-  - export PYTHON_BUILD_CACHE_PATH="/home/travis/.pyenv_cache"
-  - curl -L https://raw.githubusercontent.com/pyenv/pyenv-installer/master/bin/pyenv-installer | bash
-  - export PATH="/home/travis/.pyenv/bin:$PATH"
-  - eval "$(pyenv init -)"
-  - eval "$(pyenv virtualenv-init -)"
-  - pyenv install -s 2.7.11
-  - pyenv install -s 3.6.3
-  - pyenv global 2.7.11 3.6.3
 script:
-  - AST_COMPARE=1 yarn test -- --runInBand
+  - echo $(ruby --version)
+  - yarn run test
```

Comment:
I noticed that `AST_COMPARE=1 yarn test` currently fails, and I'm not sure if that has always been the case or not.
It shouldn't block the initial effort of getting travis enabled, as that can be fixed after PRs are being tested by Travis 🎉